api-report/driver-base.api.md

```diff
@@ -50,7 +51,7 @@ export class DocumentDeltaConnection extends EventEmitterWithErrorHandling<IDocu
     // (undocumented)
     protected earlySignalHandler: (msg: ISignalMessage | ISignalMessage[]) => void;
     // (undocumented)
-    protected emitMessages(type: string, messages: IDocumentMessage[][]): void;
+    protected emitMessages(type: string, messages: (IDocumentMessage | unknown)[][] | ISentSignalMessage[]): void;
```

Comment:
Chatted offline. I will undo the typing changes in this PR and follow up with another PR in `next` to correct the signal typings since those will be breaking changes.